main.js

```diff
@@ -258,6 +258,9 @@
         });
 
         $('#clear-search').click(clearSearch);
+        $('form.search-container').submit(function(e) {
```

Comment:
My vote would be to leave it as a form element so its picked up in any form styles that Marc may lay down.